exercises/carmen-salas/answers.js

```diff
@@ -0,0 +1,116 @@
+// Question 1
+function greet(userName) {
+  return "Good morning, " + userName + "!";
+
+}
+
+// Question 2
+function logOddNumbers() {
+  for(let i = 0; i <= 99; i++)
+  if(i % 2 ==! 0){
+    console.log(i);
+  }else{
+    
+  }
+
+}
+
+// Question 3
+function area(x, y) {
+  return (x * y); 
+}
+
+// Question 4
+function otherAngle(angle1, angle2) {
+  let angles = angle1 + angle2; 
+  return (180 - angles); 
+
+}
+
+// Question 5
+function diameterFromArea(area) {
+  let divide = (area / Math.PI);
+  let root = Math.sqrt(divide);
+  let diameter = (root * 2); 
+  return diameter;
+}
+
+
+// Question 6
+function totalPrice(price, tax) {
+  return( ((tax/100) * price) + price );
+}
+
+// Question 7
+function shortLongShort(string1, string2) {
+  if(string1.length < string2.length){
+    return (string1 + string2 + string1);
+  }else if(string1.length > string2.length){
+    return (string2 + string1 + string2);
+  }
+
+}
+
+// Question 8
+function describeAge(age) {
+  if(age <= 12){
+    return ("You're a kid.");
+  }else if(age >= 13 && age <= 17){
+    return ("You're a teenager.");
+  }else if(age >= 18 && age <= 64){
+    return ("You're an adult.");
+  }else if(64 < age){
+    return ("You're an elderly.");
+  }
+}
+
+// Question 9
+function isLeapYear(year) {
+  return(((year % 4 === 0 )&&(year % 100 != 0))||(year % 400 == 0) ? true: false);
```

Comment:
No need for the ternary operator here.
```javascript
return (year % 4 === 0 && year % 100 !== 0) || year % 400 === 0;
```

And make sure you are using strict comparison operators (`===` and `!==`).